CYBRHardeningCheck/PVWA/PVWAHardeningSteps.psm1

```diff
@@ -177,16 +177,21 @@ Function PVWA_Cryptography_Settings
 			$currentValue = Get-WebConfigurationProperty -pspath $iisPath -filter $filter -name $value
 			if($null -ne $currentValue)
 			{
-				if($currentValue.ToLower() -ne $value)
+				if($currentValue -ne 'yes')
 				{
 					$res = "Warning"
-					[ref]$refOutput.Value = "AdvancedFIPSCryptography is not properly set in PVWA Configuration. Current value: $currentValue"
+					[ref]$refOutput.Value = "AdvancedFIPSCryptography is set but does not have the correct value configured. Current value is: $currentValue"
+				}
+				else
+				{
+				$res = "Good"
+				[ref]$refOutput.Value = "AdvancedFIPSCryptography is set and has the correct value of 'yes'"
 				}
 			}
 			else
 			{
 				$res = "Warning"
-				[ref]$refOutput.Value = "AdvancedFIPSCryptography is not set in PVWA Configuration"
+				[ref]$refOutput.Value = "AdvancedFIPSCryptography key has not been set at the 'PasswordManager' level"
```

Comment:
@NathTheDude "PasswordManager"? not PVWA?

Reply:
Ha, yeah. I mean to write 'PasswordVault' :-)